staff/maryam-malek/mailchimp/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Mailchimp</title>
+</head>
+<body>
+    <header class="header">
+        <a class="header__login" href="login.html">Log In</a>
+        <div class="div__image">
+            <img class="header__image header__image--logo" src="./images/main_logo.png" alt="Main logo">
+        </div>
+        <div class="div__button">
+            <button class="button button__top grad">Sign Up Free</button>
+        </div>
+        <div class="div__nav">
+            <nav class="header__nav nav">
+                <a class="nav__element" href="pricing">Pricing</a>
+                <a class="nav__element" href="features">Features</a>
+                <a class="nav__element" href="resources">Resources</a>
+                <a class="nav__element" href="support">Support</a>
+                <a class="nav__element" href="blog">Blog</a>
+                <div class="nav__element--input">
+                    <input class="search__input" type="text" name="search" placeholder="Search...">
+                </div>
+            </nav> 
+        </div>
+        <img class="header__image header__image--monkey" src="./images/easy-email-newsletters-banner-png8.png" alt="Monkey banner">
+        <div class="header__info">
+            <img class="header__image header__info__letters" src="./images/easy-email-newsletters-text.png" alt="Easy email Newsletters">             
+            <p class="header__text">MailChimp helps you design email newsletters, share them on social networks, integrate with services you already use, and track your results. It's like your own personal publishing platform</p>
+            <button class="button button__big header__info__button grad">Sign Up Free</button>
+        </div>
+    </header>
+    <div class="div__main">
+        <div class=""></div>
+        <!-- <img class="div__main__image div__main__image--left" src="./images/noisepattern.png" alt="noise pattern"> -->
+        <main class="main">
+            <h1 class="main__title">What's New</h1>
+            <section class="main__section main__section--templates">
```

Comment:
justify content to have space between and around the main__sections

Reply:
I've organized main with grid, so I think in this case I can not do justify content with space between or arround. But I've given some space with padding.